.gitignore

```diff
@@ -1,4 +1,8 @@
 # /
+# !!! because  the ./build dir is used in the wxWidgets project as a place for persistent files !!!
```

Comment:
Sorry, I don't see any reason to add this. You can/should just create your build directory outside of the source directory or ignore it locally.